survey.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+   
+    <h1 id="title">Survey Form</h1>
+    <p id="description"> Thank you for taking the time to help us improve the platform
+        Name</p>
+    <form id="survey-form">
+      <div class = "form_control">
+        <label id="name-label">Name</label>
+        <input  type="text" id="name" required placeholder="Enter Your Name">
+      </div>
+      <div class = "form_control">
+        <label id="email-label">Email</label>
+        <input type="email" id="email" required placeholder="Enter Your email">
+      </div>

```

Comment:
Better to use semantic elements instead of <div>. [refer here](https://www.w3schools.com/html/html5_semantic_elements.asp) about HTML semantic elements.

Reply:
Good point! Please try to explain why the student should change his code, in this case, you could say to improve code clarity & structure 😉. Also, kindly try to avoid short and imperative sentences, being polite and explaining your point will make the person receiving the review feel like a suggestion instead of an order 😄. Cheers and keep coding!

> Don't be ashamed of using emojis 😀 they break the tension with ease.